src/app/app.component.ts

```diff
@@ -9,4 +13,26 @@ import { ButtonText } from '@shared/components';
 export class AppComponent {
   title = 'courses-app';
   protected readonly ButtonText = ButtonText;
+  isAuthorized!: boolean;
+
+  constructor(
+    private userStoreService: UserStoreService,
+    private authService: AuthService,
+    private coursesStoreService: CoursesStoreService,
+    private router: Router
+  ) {
+    this.authService.isAuthorized$.subscribe(
+      value => (this.isAuthorized = value)
+    );
+    this.coursesStoreService.getAll();
+    if (!this.isAuthorized) {
+      this.router.navigate(['/login']);
+    } else {
```

Comment:
Hmmm... I'm not sure about this one... see:
I'm logged in, i think i should see the course list, right?
![image](https://github.com/user-attachments/assets/91e30bbd-19e1-4e70-a1c6-103202bbccdc)

Reply:
Oh, i missed that